ibm/service/database/resource_ibm_database.go

```diff
@@ -2932,6 +3005,41 @@ func validateGroupsDiff(_ context.Context, diff *schema.ResourceDiff, meta inter
 	return nil
 }
 
+func getCpuEnforcementRatios(service string, plan string, meta interface{}, group interface{}) (cpuEnforcementRatioCeiling int, cpuEnforcementRatioMb int) {
+	var currentGroups []Group
+	defaultList, err := getDefaultScalingGroups(service, plan, "multitenant", meta)
+
+	if err != nil {
+		return 0, 0
```

Comment:
It would be better to propagate the error upward from here, rather than returning 0, 0.

Something like `return err, 0, 0`